python/private/pypi/pep508_env.bzl

```diff
@@ -64,12 +117,45 @@ _sys_platform_values = {
     "osx": "darwin",
     "windows": "win32",
 }
```

Comment:
nit: to keep better compatibility, this could be derived from the map:
```
_sys_platform_values = {
    k.partition(":")[-1]: v
    for k, v in sys_platform_select_map.items()
    if k.startswith("@platform")
}
```

The same with `platform_machine`.